File: api/Medico.Api/DB/Models/TobaccoHistory.cs
using System;

namespace Medico.Api.DB.Models
{
    public class BasePatientHistoryEntity : BaseEntity
    {
        public string Status { get; set; }

        public string Type { get; set; }

        public int? Amount { get; set; }

        public string Use { get; set; }

        public string Frequency { get; set; }

        public int? Length { get; set; }

        public string Duration { get; set; }

        public bool? Quit { get; set; }

        public int? StatusLength { get; set; }

        public string Notes { get; set; }

        public DateTime CreateDate { get; set; }

        public Guid PatientId { get; set; }

        public PatientDemographic Patient { get; set; }

        public string StatusLengthType { get; set; }
    }

    public class TobaccoHistory : BasePatientHistoryEntity
    {
    }
}

File: PatientPortal/src/app/patientChart/models/tobaccoHistory.ts
export class TobaccoHistory {
    id: string;
    status: string;
    type: string;
    amount: number;
    use: number;
    frequency: string;
    length: number;
    duration: string;
    notes: string;
    quit: boolean;
    statusLength: number;
    createDate: any;
    patientId: string;
    statusLengthType: string;

    constructor() {
        this.quit = false;
        this.createDate = new Date()    
    }
}